clearml_agent/backend_api/config/default/agent.conf

```diff
@@ -105,6 +105,10 @@
         # set to True to support torch nightly build installation,
         # notice: torch nightly builds are ephemeral and are deleted from time to time
         torch_nightly: false,
+
+        # if set to true, the agent will look for the "poetry.lock" file 
+        # in the passed current working directory instead of the repository's root directory.
+        poetry_files_from_repo_working_dir: false,
```

Comment:
Please remove trailing , (I know it appears in other places, but it's not required in HOCON syntax, and might be confusing in Python's context)